LearnositySDK/Request/Init.cs

```diff
@@ -370,12 +370,12 @@ private JsonObject generateQuestions(JsonObject output)
         /// <summary>
         /// Returns a hash value
         /// </summary>
-        /// <param name="prehash"></param>
-        /// <param name="value"></param>
+        /// <param name="prehash">String to be hashed</param>
+        /// <param name="secret">String used for encryption</param>
         /// <returns>The hashed string</returns>
-        private string hashValue(string prehash, string value )
+        private string hashValue(string prehash, string secret )
         {
-            string hash = Tools.hash(this.algorithm, prehash, value);
+            string hash = Tools.hash(this.algorithm, prehash, secret);
```

Comment:
Added summary for params changed params name to secret as per the comments given